tests/Univapay/Integration/Requests.php

```diff
@@ -377,4 +384,21 @@ public function deactivateExistingSubscriptionToken()
             $token->deactivate();
         }
     }
+
+    // use this method to mock 3DS issuer token authorization
+    public function authorizeTest3DSIssuerToken($issuerToken)
+    {
+        return wpRequests::post(
+            $issuerToken->issuerToken,
+            [
+                "Authorization" =>
+                    "Bearer " . getenv('UNIVAPAY_PHP_TEST_SECRET') . "." . getenv('UNIVAPAY_PHP_TEST_TOKEN'),
+                "Content-Type" => "application/x-www-form-urlencoded",
+            ],
+            [
+                "resource_id" => $issuerToken->payload['resource_id'],
+                "resource_type" => $issuerToken->payload['resource_type'],
+            ]
+        );
+    }
```

Comment:
What's this for again?

Reply:
I was using it for integration test with 3DS Issuer Token, but i think i over did it.
I'll remove this one